### PR TITLE
[14.0][MIG] stock_orderpoint_route: Migration to 14.0

### PR DESCRIPTION
**Pull request by dzungtran89**

### CODE REVIEW

File: stock_orderpoint_route/__manifest__.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "website": "https://github.com/OCA/stock-logistics-warehouse",
     "author": "ForgeFlow, Camptocamp, Odoo Community Association (OCA)",
     "category": "Warehouse",
-    "depends": ["stock"],
+    "depends": ["sale", "stock"],
```
Comment (grindtildeath) [inline stock_orderpoint_route/__manifest__.py]: Any reason to add this dependency?

**Review by grindtildeath — CHANGES_REQUESTED** [review]
LG, out of the extra dependency.

File: stock_orderpoint_route/__manifest__.py (hunk 1 above)
Comment (dzungtran89) [reply stock_orderpoint_route/__manifest__.py]: hello @grindtildeath , thx for your review. It seems to be a mistake, I had tested on a db with pre-installed `sale` module, and if I didn't put `sale` as dependency, the test case would have been failed due to a constraint notnull of `sale_line_warn`. I should have tested on a fresh db instead, I'll update the PR

Comment (dzungtran89) [reply stock_orderpoint_route/__manifest__.py]: @grindtildeath I pushed the update

**Review by simahawk — APPROVED** [review]

**Review by mmequignon — APPROVED** [review]

**Comment by dzungtran89** [comment]
we're checking, the test cases of module `stock_request` failed

**Comment by nilshamerlinck** [comment]
Hi @simahawk we suspect a non-deterministic issue in that case, but we have run a few builds on our side and the failure rate is low; while we investigate further, would you mind trying to merge it again?

**Comment by nilshamerlinck** [comment]
Hi @simahawk this PR will unblock the failing merge: https://github.com/OCA/stock-logistics-warehouse/pull/1262

**Pull request merged**